test/decoders/test_simple_video_decoder.py

```diff
@@ -322,11 +322,7 @@ def test_get_frame_displayed_at(self):
 
     def test_get_frame_displayed_at_h265(self):
         decoder = SimpleVideoDecoder(H265_VIDEO.path)
-        # Note that for H265, FFMPEG's seeking is not precise. Even though we ask to
-        # seek with a max_ts=0.5, FFMPEG will seek beyond that point.
-        # TODO: Revert use frame5 in the test below once it's fixed upstream:
-        # https://trac.ffmpeg.org/ticket/11137
-        ref_frame6 = H265_VIDEO.get_frame_by_name("frame000006")
+        ref_frame6 = H265_VIDEO.get_frame_by_name("frame000005")
```

Comment:
Now that the comment was removed, it's not immediately obvious why this test exists. It might be worth clarifying its purpose by adding e.g.

```
# Non-regression test for https://github.com/pytorch/torchcodec/issues/179
```